smokey/src/main/java/frc/robot/control/AutonomousChooser.java

```diff
@@ -124,7 +124,11 @@ public Command getScoreRoutine(Pose2d NodePosition){
         if(this.subsystems.getArmSubsystem() != null) {
             SequentialCommandGroup armSequence = new SequentialCommandGroup();
             armSequence.addCommands(new ArmToReferencePositionCommand(subsystems.getArmSubsystem()));
-            armSequence.addCommands(new ArmToLocationCommand(subsystems.getArmSubsystem(), ArmLocation.ARM_HIGH_SCORE));
+            armSequence.addCommands(
+                new ArmToLocationCommand(
+                    subsystems.getArmSubsystem(),
+                    ArmLocation.ARM_HIGH_SCORE,
+                    subsystems.getManualInputInterfaces()));
```

Comment:
should we have a command at the start of auto that explicitly sets the game piece to cube? (so that we aren't relying on the default to still be set?)

Reply:
good idea, I will update ... also this block changed with the merge

Reply:
made the change